docs/learning/README.md

```diff
@@ -98,8 +126,15 @@ This is a general set of technical links and recommended reading our team has fo
 - [How Schnorr signatures may improve Bitcoin](https://medium.com/cryptoadvance/how-schnorr-signatures-may-improve-bitcoin-91655bcb4744)
   - A gentle introduction to Schnorr signatures
 
+### P2P
+
+- [Eclipsing Ethereum Peers with False Friends](https://arxiv.org/pdf/1908.10141.pdf)
+  - A detailed explanation of how Kademlia (DHT for P2P networks) works, accompanying a deep dive into peer management and peer discovery in Geth, with the goal of outlining several attack vectors and their countermeasures.
+
 ### Blogs
 
+#### Blogs - Specific Articles
```

Comment:
Blogs prefix is unnecessary - also, this isn't a pattern you follow anywhere else